### PR TITLE
Create user guide for module setup

### PR DESCRIPTION
**Pull request by sinukaarel**
Created new module images.
Added `userguide.md` to have a reference when updating the plugin in Magento marketplace.

### CODE REVIEW

File: userguide.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# USER GUIDE
+
+## Smaily E-Commerce extension for Magento 2.0+
+>[Smaily](https://smaily.com/) is an intentionally simple tool for sending beautiful email newsletters. 
+
+## Purchase / Download
+
+1. You can purchase **Smaily E-Commerce extension for Magento 2.0+** from [Magento marketplace](https://marketplace.magento.com/smaily-smailyformagento.html) or download it from our [github repository](https://github.com/sendsmaily/smaily-magento-extension).
```
Comment (kaittodesk) [inline userguide.md]: `github repository` should be written as `GitHub repository`.

File: userguide.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+# USER GUIDE
+
+## Smaily E-Commerce extension for Magento 2.0+
+>[Smaily](https://smaily.com/) is an intentionally simple tool for sending beautiful email newsletters. 
+
+## Purchase / Download
```
Comment (kaittodesk) [inline userguide.md]: Maybe it would be more clear if this heading is `Installing extension`?

File: userguide.md (hunk 1 above)
Comment (kaittodesk) [inline userguide.md]: Is it worth to mention that the extension is free or use `You can get` instead of `You can purchase`?

File: userguide.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+1. You can purchase **Smaily E-Commerce extension for Magento 2.0+** from [Magento marketplace](https://marketplace.magento.com/smaily-smailyformagento.html) or download it from our [github repository](https://github.com/sendsmaily/smaily-magento-extension).
+
+2. To install our extension follow the [Magento extension installing guide](https://docs.magento.com/marketplace/user_guide/buyers/install-extension.html)
+
+## Account validation
```
Comment (kaittodesk) [inline userguide.md]: Perhaps change the title to `Connecting to Smaily account`?

File: userguide.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+5. We recommend to **use CATPCHA** to prevent bots from polluting your newsletter subscribers list. You can use two options - **Magento`s text-based CAPTCHA or Google reCAPTCHA**.
+
+![Newletter subscription form](assets/newsletter_subscription_form.png)
+
+## Subscribers synchronization
+
+6. Enable automatic subscribers synchronization feature under **Subscribers synchronization** section. 
+
+7. There is an option to **import additional fields** available from store into Smaily to personalize newsletter emails. **Frequency** can be set from once a week up to once every 4 hours.
```
Comment (kaittodesk) [inline userguide.md]: `from once a week up to once every 4 hours` can be a bit vague on which options are available. One could assume that any value between that range is valid. What if we explicitly define what options are available? Would that make sense?

File: userguide.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Abandoned cart emails
+
+8. Enable abandoned cart emails feature under **Abandoned Cart** section to send cart reminder emails to store customers.
+
+9. You need to create *form submitted* workflow in Smaily prior to activating this feature. You can follow our [creating automation workflows tutorial](http://help.smaily.com/en/support/solutions/articles/16000092458-creating-automation-workflows).
+
+10. After creating automation in Smaily you can find this automation under **Autoresponder ID**.
+
+11. You can **chose timing** when cart is considered abandoned form 20 minutes up to 12 hours.
```
Comment (kaittodesk) [inline userguide.md]: `chose` means a different thing. It should be `choose` :)

File: userguide.md (hunk 2 above)
Comment (sinukaarel) [reply userguide.md]: Maybe `Installation` then?

Comment (kaittodesk) [reply userguide.md]: Yeah, we could go with that also.

File: userguide.md (hunk 4 above)
Comment (sinukaarel) [reply userguide.md]: Actually I would omit these values. Whoever reads this guide wants to get the first short introduction or overview of what is possible.
> The synchronization **frequency can be adjusted**

Also, the abandoned cart timing option could be shortened.
>  You can **choose timing** when cart is considered abandoned.

Comment (kaittodesk) [reply userguide.md]: That might be true for some users, but others would still like to know about the available frequency or timing options.

File: userguide.md (hunk 1 above)
Comment (sinukaarel) [reply userguide.md]: Changed to GitHub https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2

File: userguide.md (hunk 2 above)
Comment (sinukaarel) [reply userguide.md]: Done  https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2

File: userguide.md (hunk 1 above)
Comment (sinukaarel) [reply userguide.md]: Changed to `you can get` https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2

File: userguide.md (hunk 3 above)
Comment (sinukaarel) [reply userguide.md]: Done https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2

File: userguide.md (hunk 4 above)
Comment (sinukaarel) [reply userguide.md]: Added the whole list https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2

File: userguide.md (hunk 5 above)
Comment (sinukaarel) [reply userguide.md]: Fixed my typo https://github.com/sendsmaily/smaily-magento-extension/pull/34/commits/ce76c4eb0d9c5088068b7af75c6bdcf4b874b3d2

**Pull request merged**